src/ucp/amo/nb_amo.c

```diff
@@ -0,0 +1,279 @@
+/**
+* Copyright (C) Mellanox Technologies Ltd. 2001-2015.  ALL RIGHTS RESERVED.
```

Comment:
this is a new file. I don't think you need MLNX copyright, unless you did copy-n-past from other file.